### ISSUE TITLE
Bauh is attempting to downgrade Falkon after Mint's updater updated it

### ISSUE THREAD

**Issue by Templayer** [vinifmor/bauh#326]
See the version string:

![image](https://user-images.githubusercontent.com/11728067/237060702-d2f6e79f-4002-4325-b285-6cc5e0868013.png)

![image](https://user-images.githubusercontent.com/11728067/237060740-93c89110-af91-46e8-96db-2146329c6c6c.png)

 Static hostname: JB-PC
       Icon name: computer-laptop
         Chassis: laptop
Operating System: Linux Mint 21.1                   
          Kernel: Linux 5.19.0-41-generic
    Architecture: x86-64
 Hardware Vendor: Micro-Star International Co., Ltd.
  Hardware Model: Bravo 17 A4DDR

**Comment by Templayer** [comment]
The Mint updater meanwhile has this now:

![image](https://user-images.githubusercontent.com/11728067/237061775-74cb8dcf-c2dc-4a07-934b-0467775b0b83.png)

This might be some sort of Falkon issue, as this is the first time I see such a problem.

**Comment by vinifmor** [comment]
Hi @Templayer , if it happens again, please, paste the output of `flatpak update` here so I can have a look. Thank you for reporting the problem...

**Comment by Templayer** [comment]
> Hi @Templayer , if it happens again, please, paste the output of `flatpak update` here so I can have a look. Thank you for reporting the problem...

Yeah, this might just be a matter of Falkon repo maintainers screwing up and then fixing it.
